src/features/Table/Row/index.tsx

```diff
@@ -3,15 +3,25 @@ import Cell from '../Cell';
 export declare interface RowInterface {
   data: string[]
   head?: boolean
+  isClickable?: boolean
+  onClick?: () => void
 }
+const Row = ({ data, head = false, isClickable = false, onClick }: RowInterface): JSX.Element => {
+  const handleOnClick = (): void => {
+    if (onClick != null) {
+      onClick();
+    }
+  };
 
-const Row = ({ data, head = false }: RowInterface): JSX.Element => {
   return (
+    <>
     <tr className='border-gray-100 border-b last:border-b-0'>
+      <td>{isClickable ? <input type="checkbox" onClick={handleOnClick} /> : null}</td>
```

Comment:
In type script, there are many syntaxes and ways to write a statement. All should be correct unless if it is throwing errors, making conflicts with project or if it does not work. 

When you suggest changing a syntax for a working statement. You should be descriptive of the reason of doing this, what are the benefits, and what are the impact if it left unchanged.

Reply:
This is not typescript but React. In React/jsx, using `&&` provides a cleaner way to present the conditional rendering since we don't care for the alternate element. It also reduced the amount of code we have to write. This is more a recommendation to have a limited amount of code.